.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
This is great.  How about pulling in the rakefile target and adding it to travis as well?